addons/event_booth_sale/models/product_product.py

```diff
@@ -0,0 +1,37 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Product(models.Model):
+    _inherit = 'product.product'
+
+    is_event_booth = fields.Boolean(
+        compute="_compute_is_event_booth",
+        compute_sudo=True,
+        search="_search_is_event_booth",
+    )
+
+    def _compute_is_event_booth(self):
```

Comment:
You can create a service product and then link it, since the domain on events is now products of service type.
Nevertheless, I kinda agree that the detailed type changes are not the best part of this spec :D (fp-request though :c)

Reply:
I suggest to postpone to next freeze so that we calmy check and test everything, especially LNA is waaaay busy those days. Don't think people will die if they wait for 2 months, will they ?

Reply:
Well, our PO is on holidays for the freeze, and he didn't test the task either :D.
We'll try to merge the true back 2 basics change of this task for the freeze, and the rest will probably wait for next freeze (and more debates/testing with PO's/FP)

Reply:
Indeed, don't think we have to rush model changes in intermediate saas, does not seem to be the critical point right now.